tools/links.py

```diff
@@ -64,6 +64,7 @@
     "ansys-tools-local-product-launcher": "https://local-product-launcher.tools.docs.pyansys.com/version/stable",  # noqa: E501
     "ansys-tools-path": "https://path.tools.docs.pyansys.com/version/stable",
     "ansys-tools-protoc-helper": None,
+    "ansys-tools-visualization-interface": "https://visualization-interface.tools.docs.pyansys.com/",  # noqa: E501

```

Comment:
You should also add the repo to the milestone.py script or workflow. I can't remember which one specifically